python/ray/tune/BUILD

```diff
@@ -313,6 +313,8 @@ py_test(
 py_test(
     name = "test_trial_scheduler_pbt",
     size = "large",
+    # This may be a bit exaggerated but just to see if tests pass
+    timeout = "eternal",
```

Comment:
Nit: Could upgrade the size to `"enormous"`.
```suggestion
    size = "enormous",
```
https://docs.bazel.build/versions/main/be/common-definitions.html#common-attributes-tests
<img width="727" alt="Screen Shot 2021-12-15 at 6 19 42 PM" src="https://user-images.githubusercontent.com/7699919/146295953-7b263974-e66c-418e-b064-882863dcbbbc.png">

Reply:
Even though these both end up with the same timeout, it's not like the size of the test is actually increasing, so wouldn't it make more sense just to explicitly increase the timeout?

Reply:
I've always equated size with timeout (haven't paid much mind to the peak RAM estimate), but you make a good point. Giving the timeout separately seems better, especially given that this test shouldn't have that large if a peak memory utilization. Might even be better to remove the `size="large"` argument since I doubt that the peak memory utilization goes above 100 MB.

Reply:
Hmm technically that would be the optimal setup, but I'd still lean towards keeping `size="large"` in this PR because:

1. The pattern in this repo is to use `size` to dictate timeout.
2. This is just an override for test purposes that should probably be removed later (and use `size="large"` for timeout).
3. Other developers may not understand what's going on since this is the only test without `size`.